UnitsNet/UnitDefinitions/Entropy.json

```diff
@@ -0,0 +1,46 @@
+﻿{
+  "Name": "Entropy",
+  "BaseUnit": "JoulePerKelvin",
+  "XmlDoc": "",
```

Comment:
Did you forget to add the xmldoc?

I'm adding in this text based on a few sections in the wiki:

> Entropy is an important concept in the branch of science known as thermodynamics. The idea of "irreversibility" is central to the understanding of entropy.  It is often said that entropy is an expression of the disorder, or randomness of a system, or of our lack of information about it. Entropy is an extensive property. It has the dimension of energy divided by temperature, which has a unit of joules per kelvin (J K−1) in the International System of Units.